package.json

```diff
@@ -6,15 +6,15 @@
   "scripts": {
     "dev": "grunt",
     "build": "grunt release",
-    "test": "jest -u --coverage && python manage.py test",
-    "test:docker": "jest -u && docker-compose run web python manage.py test",
-    "collectstatic": "python manage.py collectstatic --no-input",
+    "test": "jest -u --coverage && python3 manage.py test",
```

Comment:
I'd rather this stayed as `python` ... up to you to point the `python` command to run your preferred version of python. It's future proof. Would rather not have to change this to python4 whenever that version gets out.

https://askubuntu.com/questions/320996/how-to-make-python-program-command-execute-python-3

Reply:
Darn, couldn't merge this in until you also approve @edwinmp. As @Napho said in Slack, it's not much of a concern because python2 is incompatible with python3 which will be incompatible with python4. In fact, this incompatibility is leading to my local install complaining when you try and run Python3 Django with the system python3.